lib/module.js

```diff
@@ -250,7 +250,7 @@ Module._resolveLookupPaths = function(request, parent) {
 
   // make sure require('./path') and require('path') get distinct ids, even
   // when called from the toplevel js file
-  if (parentIdPath === '.' && id.indexOf('/') === -1) {
+  if (parentIdPath === '.' && id.includes('/')) {
```

Comment:
@tejasmanohar It looks like this could use a test. It is not absolutely necessary from this PR's point of view, but a test which covers this case would be awesome :-)